sqlitecluster/SQLiteNode.cpp

```diff
@@ -1399,11 +1402,6 @@ bool SQLiteNode::update(uint64_t& nextActivity) {
 
                                 // By returning 'true', we update the FSM immediately, and thus evaluate whether or not
                                 // we need to wait for quorum.  This keeps all the quorum logic in the same place.
-                                if (STimeNow() > nextActivity) {
-                                    SINFO("Timeout reached while processing (transaction) commands. Exceeded by "
-                                          << (STimeNow() - nextActivity) << "us");
-                                    return false;
-                                }
```

Comment:
We don't want to return false here, we want to return `true`, which causes the `_currentCommand` that we just set to be evaluated for quorum. If we return false here, and there's no new activity on the network, we'll sit for a second before evaluating whether or not this command can be committed. Instead, we've updated the logic there to return `false` after we *commit* a command, rather than returning false here after we begin a transaction.